src/consensus/raft/raft.h

```diff
@@ -624,7 +626,7 @@ namespace raft
         {
           LOG_DEBUG_FMT(
             "Recv append entries to {} from {} but our log at {} has the wrong "
-            "term (ours: {}, theirs: {})",
+            "previous term (ours: {}, theirs: {})",
```

Comment:
nit: view

Reply:
`raft.h` still uses the Raft-specific term-inology (no pun intended) for both comments and logs. I think we should address this separately if we want to? The `raft_consensus.h` adapter provides the translation between both.